testcases/MachineConfig.py

```diff
@@ -327,6 +341,28 @@ def CecSetup_lmb(self):
                                    (self.system_name, self.lpar_name, self.lpar_prof))
             time.sleep(5)
 
+    def CecHugepageSetup_16gb(self):
+
+        self.current_hgpg = self.cv_HMC.get_16gb_hugepage_size()
+        if int(self.current_hgpg[0]) != 0:
+            log.info("System is already configured with 16gb Hugepage")
+        else:
+            self.cv_HMC.poweroff_system()
```

Comment:
if CEC and huge page setting called in single flow can we do this operation once like
1- poweroff
2-
 >> set LMB size 
 >> set hugepage
3- Power on 

easy way is that may be rename ceclmb function and accommodate request in if else condition